Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -518,24 +514,32 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
     return clickableViews;
 }
 
-- (BannerSize)adSizeFromAdFormat:(MAAdFormat *)adFormat
+- (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
+                          parameters:(id<MAAdapterParameters>)parameters
 {
-    if ( adFormat == MAAdFormat.banner )
+    BOOL isAdaptiveBanner = [parameters.localExtraParameters al_boolForKey: @"adaptive_banner"];
+    NSNumber *customWidth = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_width"] ? : 0;
+    NSNumber *customHight = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_height"] ? : 0;
+
+    if (!isAdaptiveBanner && customWidth && customWidth) {
```

Comment:
Can u move the line 521 and 522 inside here. and since u ave a default value as 0 which our sdk support u dont need to check customWidth && customWidth in if block.

Reply:
Good catch. Thanks!
Updated.